bin/bb/cmd/var.go

```diff
@@ -6,3 +6,18 @@ import "go.uber.org/zap"
 var (
 	logger *zap.Logger
 )
+
+const dsnUsage = `Database connection string.
+
+DSN format:
+  DRIVER://USERNAME[:PASSWORD]@HOST[:PORT]/[DATABASE][?PARAM=VALUE&...&PARAM=VALUE]
+
+Drivers:
+  mysql
+  postgresql
+
+Examples:
```

Comment:
nit: put mysql example first to follow the Driver's order.

And mysql is more popular so it makes sense to put it first.